### PR TITLE
API added

### PR DESCRIPTION
**Pull request by PaterSantyago**

### CODE REVIEW

File: src/Lykke.Quintessence.Api/ApiStartupBase.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        protected override bool IsDebug
+            => ApiStarter.IsDebug;
+
+        protected override string ServiceName
+            => "Api";
+
+        protected override void ConfigureFluentValidation(
+            FluentValidationMvcConfiguration configuration)
+        {
+            base.ConfigureFluentValidation(configuration);
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Api/ApiStartupBase.cs:23]: What is the desired flow to override validators?

Comment (PaterSantyago) [reply src/Lykke.Quintessence.Api/ApiStartupBase.cs:23]: Let's leave it intact and refactor in future release

File: src/Lykke.Quintessence.Api.DomainServices/Strategies/DefaultCalculateGasAmountStrategy.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                    }
+                }
+                else
+                {
+                    return GasAmountCalculationResult.Error("Gas amount can not be estimated.");
+                }
+            }
+            else
+            {
+                return GasAmountCalculationResult.GasAmount(21000);
```
Comment (cassade) [inline src/Lykke.Quintessence.Api.DomainServices/Strategies/DefaultCalculateGasAmountStrategy.cs:87]: This constant [theoretically] may be changed with fork or software update.

**Review by cassade — APPROVED** [review]

File: src/Lykke.Quintessence.Api.DomainServices/Strategies/DefaultCalculateGasAmountStrategy.cs (hunk 2 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Api.DomainServices/Strategies/DefaultCalculateGasAmountStrategy.cs:87]: Created an issue, I'll fix it in next version

**Pull request merged**